Accessibility/Accessibility/Cells/CategoryCollectionViewCell.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  CategoryCollectionViewCell.swift
+//  Accessibility
+//
+//  Created by Nicole Souvenir on 1/23/19.
+//  Copyright © 2019 Nicole Souvenir. All rights reserved.
+//
+
+import UIKit
+import URBNSwiftyConvenience
+
+class CategoryCollectionViewCell: UICollectionViewCell {
```

Comment:
Mark `final` if you don't plan on subclassing

Reply:
fixed